jib-core/src/main/java/com/google/cloud/tools/jib/api/KeepOriginalModificationTimeProvider.java

```diff
@@ -0,0 +1,26 @@
+package com.google.cloud.tools.jib.api;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.Instant;
+
+/** Modification time provider which returns original file modification time */
+public class KeepOriginalModificationTimeProvider implements ModificationTimeProvider {
```

Comment:
I'll need to decide whether we want to have this in jib-core. Maybe in jib-plugins-common. Particularly, the method throwing a runtime exception is not an ideal behavior as a library implementation.